libgadget/density.c

```diff
@@ -365,7 +396,7 @@ density_ngbiter(
 
     if(All.WindOn) {
         if(winds_is_particle_decoupled(other))
-            if(!(I->DelayTime > 0))	/* if I'm not wind, then ignore the wind particle */
+            if(!(I->Type == 0 && I->DelayTime > 0))	/* if I'm not wind, then ignore the wind particle */
```

Comment:
So Bh were randomly not getting a density update?

Reply:
BH were randomly including wind particles in their density estimation

Reply:
Oops. Now BH always include wind particles in there density, which doesn't sound sensible but was the way it was supposed to be, I think.